app/config/AppConfig.scala

```diff
@@ -40,6 +40,9 @@ class AppConfig @Inject()(
     .getOptional[String]("mongodb.uri")
     .map(uri => MongockConfig(uri))
 
+  private lazy val contactBaseUrl = servicesConfig.baseUrl("contact-frontend")
```

Comment:
Could you rename it to `contactFrontendBaseUrl`? So it's in th esame convention as the field below.